testing/ncids-css-testing/stories/components/nci-autocomplete/nci-autocomplete.scss

```diff
@@ -0,0 +1,129 @@
+@import '@nciocpl/ncids-css/scss/lib/normalize';
+@import '@nciocpl/ncids-css/scss/packages/required';
+@import '@nciocpl/ncids-css/scss/packages/global';
+@import '@nciocpl/ncids-css/scss/components/nci-header/nci-extended';
```

Comment:
Can you please make some imports/stories that are autosuggest without a header? Actually, there should be NO autocomplete stories with a header -- those stories should be with the header if needed.